org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm

```diff
@@ -310,7 +314,7 @@ class CentralController:
 
     def __init__(self):
         self.concState = $!{rootState.getName()}_State(None, None)
-        self.possibleEvents = {#if(${allEvents.size()} > 0)"${allEvents.get(0).getModifiedName()}"#foreach($event in ${allEvents.subList(1, ${allEvents.size()})}), "$!{event.getModifiedName()}"#end#end}
+        self.possibleEvents = {#if(${allEnvEvents.size()} > 0)"${allEnvEvents.get(0).getModifiedName()}"#foreach($event in ${allEnvEvents.subList(1, ${allEncEvents.size()})}), "$!{event.getModifiedName()}"#end#end}
```

Comment:
Could be more readable if we move this logic into Java or add some indentation? But I am fine leaving it here.